hoomd/md/NeighborList.cc

```diff
@@ -438,22 +440,14 @@ void NeighborList::updateRList()
             throw std::invalid_argument("given r_cut_matrix is not the right size");
             }
 
-        if (matrix == 0)
-            {
-            // copy the first matrix as a starting point
-            memcpy(h_r_cut.data, h_consumer_r_cut.data, sizeof(Scalar)*m_r_cut.getNumElements());
-            }
-        else
+        // take the maximum
+        for (unsigned int i=0; i < m_pdata->getNTypes(); ++i)
             {
-            // take the maximum
-            for (unsigned int i=0; i < m_pdata->getNTypes(); ++i)
+            for (unsigned int j=0; i < m_pdata->getNTypes(); ++i)
```

Comment:
Should this be a `j`?
```suggestion
            for (unsigned int j=0; j < m_pdata->getNTypes(); ++j)
```

Reply:
Good catch.